src/auth-service/controllers/role.controller.js

```
@@ -570,347 +435,237 @@ const createRole = {
+              await rolePermissionsUtil.enhancedUnAssignUserFromRole(
+                request,
+                next
+              );
+              result = await rolePermissionsUtil.enhancedAssignUserToRole(
+                request,
+                next
+              );
+              break;
+          }
+
+          if (result && result.success) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)